test/concerns/hideable_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class HideableTest < Capybara::Rails::TestCase
```

Comment:
@nkumeiko, why do you use `Capybara::Rails::TestCase` class and not ActiveSupport::TestCase?